.formatter.exs

```diff
@@ -1,6 +1,6 @@
 [
   inputs: [
-    "{lib,examples,test,config}/**/*.{ex,exs}",
+    "{lib,examples,test,config,playlist_generation}/**/*.{ex,exs}",
```

Comment:
Do we still need `playlist_generation` here?

Reply:
Absolutely not, I forgot to erase it from formatter config.